resources/mediaplayer/src/main/java/org/mobicents/media/server/impl/resource/mediaplayer/audio/wav/WavTrackImpl.java

```diff
@@ -64,8 +66,9 @@
     private final static byte[] factBytes = new byte[] { 0x66, 0x61, 0x63, 0x74 };
     private byte paddingByte = PCM_PADDING_BYTE;
 
-    public WavTrackImpl(URL url) throws UnsupportedAudioFileException, IOException {
-        inStream = url.openStream();
+    @Inject
```

Comment:
Can you please remove injection from this class?
I don't want the project to be fully dependent on guice. As rule of thumb we Guice for solving complicated dependencies during bootstrapping only.

Maybe a workaround is to have the player receiving a "TrackFactory" for example.